<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWorkerWindow</class>
 <widget class="QDialog" name="AddWorkerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="addWorker">
     <property name="text">
      <string>Добавить нового сотрудника</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="personalInformation">
        <property name="title">
         <string>Личная информация</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="secondName">
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="secondNameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="firstName">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="firstNameEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="middleName">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="middleNameEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="Account">
        <property name="title">
         <string>Данные об аккаунте</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="login">
           <property name="text">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="loginEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="password">
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="passwordEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="isDispatcher">
           <property name="text">
            <string>Диспетчер</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="isAdmin">
           <property name="text">
            <string>Администратор</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="isWorker">
           <property name="text">
            <string>Рабочий</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addWorker</sender>
   <signal>clicked()</signal>
   <receiver>AddWorkerWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
